### PR TITLE
Test cases for check_bipartite_graph_bfs

### PR DESCRIPTION
**Pull request by RaymondDashWu**
### Describe your change:
Contributes to #9943 . For maintainers: The automated tester is saying [L65](https://github.com/TheAlgorithms/Python/pull/10688/files#diff-8371f7cda63c896b2318d7c01f080230a91b524e7cc869dec23e9aa568a6106bR65) needs tests but I'm not sure what it's asking for.

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?
* [X] Add test cases

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [X] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

File: graphs/check_bipartite_graph_bfs.py [hunk 1]
```diff
@@ -10,6 +10,24 @@
...
+    ... )
+    False
+    >>> check_bipartite(
+    ... {0: [4], 1: [], 2: [4], 3: [4], 4: [0, 2, 3]}
+    ... )
+    True
+    >>> check_bipartite(
+    ... {0: [1, 3], 1: [0, 2], 2: [1, 3], 3: [0, 2], 4: [0]}
+    ... )
+    False
```
Comment (cclauss) [inline graphs/check_bipartite_graph_bfs.py:29]: Please add some negative values, floating point values, and some strings.
Also,
`{7: [1, 3], 1: [0, 2], 2: [1, 3], 3: [0, 2], 4: [0]}`
`{0: [1, 3], 1: [0, 2], 2: [1, 3], 3: [0, 2], 9: [0]}`

Comment (RaymondDashWu) [reply graphs/check_bipartite_graph_bfs.py:29]: thanks for the additional test cases! I'll have to refactor the code a bit as it assumes that there's keys for all values from 0 to n. Will add different data types as well.

Comment (cclauss) [reply graphs/check_bipartite_graph_bfs.py:29]: It is OK to fail if the input is not 1 to n or if other garbage data is provided.  We just want to ensure the code does safe things in the face of garbage data.  Raising a ValueError is safe.

Comment (RaymondDashWu) [reply graphs/check_bipartite_graph_bfs.py:29]: Understood. Thanks for the clarification!

**Comment by RaymondDashWu** [comment]
The only failing error deals with an issue mentioned in https://github.com/TheAlgorithms/Python/pull/10717. I'll be moving on to another PR.

**Review by cclauss — APPROVED** [review]

**Pull request merged**